### PR TITLE
Fix postgres Datadog integration

### PR DESCRIPTION
**Pull request by sauloperez**
Due to two issues, most of the Postgres Datadog graphs show empty now, especially those related to indexes.

The first is one is that we were creating neither the function nor the view Datadog needs because no database was passed to psql. The second one is that we manually need to specify the db relations we want to track in the Datadog's config file.

### CODE REVIEW

File: roles/datadog/tasks/pg_stats.yml [hunk 1]
```diff
@@ -76,9 +76,16 @@
...
-    CREATE VIEW pg_stat_activity_dd AS SELECT * FROM pg_stat_activity();
-    grant SELECT ON pg_stat_activity_dd to datadog;"
+    CREATE VIEW pg_stat_activity_dd AS SELECT * FROM pg_stat_activity();"
   become: yes
   become_user: postgres
   when: stat_views_exist.stdout.find('0 rows') != -1
   notify: restart postgres
+
+- name: grant select to datadog
+  command: |
```
Comment (enricostano) [inline roles/datadog/tasks/pg_stats.yml]: Do you think that the following module wold make this task idempotent?

https://docs.ansible.com/ansible/latest/modules/postgresql_privs_module.html

Comment (sauloperez) [reply roles/datadog/tasks/pg_stats.yml]: Nice @enricostano !

Comment (sauloperez) [reply roles/datadog/tasks/pg_stats.yml]: We are also looking forward to https://docs.ansible.com/ansible/latest/modules/postgresql_query_module.html#postgresql-query-module which got included in 2.8

**Comment by sauloperez** [comment]
Ready in Katuma production. The data we start gathering today and tomorrow will be invaluable.

screencapture-app-datadoghq-dashboard-ki8-7zn-7gr-postgres-overview-cloned-2019-05-26-12_35_10

:sunglasses:

File: roles/datadog/tasks/pg_stats.yml (hunk 1 above)
Comment (enricostano) [reply roles/datadog/tasks/pg_stats.yml]: Me too! :grimacing:

**Comment by sigmundpetersen** [comment]
2 approvals, can we move this along?

**Comment by sauloperez** [comment]
Is it fine if we provision UK and FR this week @Matt-Yorkley

**Comment by Matt-Yorkley** [comment]
Sure :ok_hand:

**Pull request merged**